inc/languages/english/usercp.lang.php

```diff
@@ -291,7 +291,8 @@
 $l['date_uploaded'] ="Posted";
 $l['no_attachments_selected'] = "You did not select any attachments to delete.";
 $l['attachments_deleted'] = "The selected attachments have been successfully deleted.<br />You will be now redirected to the attachments manager.";
-$l['attachments_usage_quota'] = "- Using {1} of {2} ({3}) in {4} Attachments";
+$l['attachments_usage_quota'] = "- Using {1} of {2} in {3} Attachments";
+$l['attachments_usage_percent'] = "( {1}%)";
```

Comment:
Is that space intentional ?

Reply:
Yep. While the % is being appended to the usage size ...
Just moved it out of usercp.php, as shade instructed to avoid hardcoding and placed in language.